src/app/workshops-list/workshops-list.component.html

```diff
@@ -3,23 +3,22 @@
         <article class="entity" *ngFor="let data of workshops;">
             <div class="inner" (click)="loadWorkshopDetails(data.workshopId, data.name)">
                 <img class="image" [src]="data.imageLink" alt="workshop image" />
-                <h1 class="name">
+                <h1 class="name truncate" title="{{ data.name }}">
                     {{ data.name }}
                 </h1>
                 <hr>
                 <div class="location">
                     <i class="fa fa-map-marker" aria-hidden="true"></i>
-                    <!-- <span>  {{data.location.city}},</span> -->
-                    <!-- <span> {{data.location.state}},</span> -->
-                    <!-- <span> {{data.location.country}} </span> -->
+                    <span>  {{data.location?.city}},</span>
```

Comment:
we need better logic here. Opening a bug for this so we don't forget. Fine for now.